server.js

```diff
@@ -27,7 +31,50 @@ app.get('/', (req, res) => {
   res.send('Hello world')
 })
 
+// The entire json
+app.get('/book', (req, res) => {
+  res.json(booksData)
+})
+
+// Get one book from ID
+app.get('/books/:id', (req, res) => {
+  const bookId = req.params.id
+  const book = booksData.find((item) => item.bookID === +bookId)
+  if (book) {
+    res.json(book);
+  } else {
+    res.status(404).send("No book found")
+  }
+})
+
+// Get language filterd on language_code
+app.get('/language/:language', (req, res) => {
+  const language = req.params.language
```

Comment:
you could also destructure language from req.params.
`const { language } = req.params`